README.md

```diff
@@ -32,7 +32,7 @@ Environment variables include:
   - **PULL_REQUEST_BRANCH**: the branch to issue the pull request to. Defaults to master.
   - **PULL_REQUEST_BODY**: the body for the pull request (optional)
   - **PULL_REQUEST_TITLE**: the title for the pull request  (optional)
-  - **MAKE_DRAFT_PR**: should the pull request be a draft PR? (optional; defaults to `false`)
+  - **MAKE_DRAFT_PR**: should the pull request be a draft PR? (optional; unset defaults to `false`)
 
```

Comment:
Perfecto! This one little fix here (`MAKE_DRAFT_PR` -> `PULL_REQUEST_DRAFT` and we are ready for merge.

Reply:
<img width="136" alt="image" src="https://user-images.githubusercontent.com/9143563/57961875-d68e0480-78c6-11e9-95d4-0a92bf14e90f.png">

Smh just hadn't saved that change oops 🤦‍♂

Reply:
No worries, I've done that too :)